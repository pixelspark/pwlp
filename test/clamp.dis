0000.	11	PUSHB	[14]
0002.	11	PUSHB	[01]
0004.	21	PEEKB	1
0005.	10	PUSHB	0
0006.	21	PEEKB	1
0007.	21	PEEKB	1
0008.	8a	BINARY	LT
0009.	50	JZ	to 15
0012.	01	POP	1
0013.	fc	SPECIAL	swap
0014.	01	POP	1
0015.	60	JNZ	to 19
0018.	02	POP	2
0019.	11	PUSHB	[0a]
0021.	21	PEEKB	1
0022.	21	PEEKB	1
0023.	88	BINARY	GT
0024.	50	JZ	to 30
0027.	01	POP	1
0028.	fc	SPECIAL	swap
0029.	01	POP	1
0030.	60	JNZ	to 34
0033.	02	POP	2
0034.	74	UNARY	SHL8
0035.	86	BINARY	OR
0036.	21	PEEKB	1
0037.	74	UNARY	SHL8
0038.	74	UNARY	SHL8
0039.	86	BINARY	OR
0040.	21	PEEKB	1
0041.	74	UNARY	SHL8
0042.	74	UNARY	SHL8
0043.	74	UNARY	SHL8
0044.	86	BINARY	OR
0045.	e3	USER	set_pixel
0046.	01	POP	1
0047.	e4	USER	blit
0048.	01	POP	1

