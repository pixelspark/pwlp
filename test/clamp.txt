a=20;
set_pixel(1, clamp(a, 0, 10), a, a);
blit;