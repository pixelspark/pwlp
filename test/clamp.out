000000 0a1414 000000 000000 000000 000000 000000 000000 000000 000000 
